cfme/infrastructure/pxe/db.py

```diff
@@ -0,0 +1,15 @@
+from base import PXEBase
+from utils.appliance import ViaDB
+
+
+class PXEServerDB(PXEBase):
+    @PXEBase.exists.implemented_for(ViaDB)
+    def existse(self):
+        """
+        Checks if the PXE server already exists
+        """
+        return self.impl.session.query(
```

Comment:
So... why are we referring to `ViaDB` as `self.impl. instead of directly going to `ViaDB` or just `DB`? The implementation is for sure going to be `DB` so why hide it?